Microsoft.Toolkit.Mvvm/ComponentModel/ObservableObject.cs

```diff
@@ -7,6 +7,14 @@
 // This file is inspired from the MvvmLight library (lbugnion/MvvmLight),
 // more info in ThirdPartyNotices.txt in the root of the project.
 
+// ================================== NOTE ==================================
```

Comment:
Guess there's no way around this as their structure is slightly different, eh?

Reply:
Yeah, unfortunately there's some differences over all the file that make it impossible to just reuse the same file - we'd have to manually parse it and then exclude all the unwanted members and individual statements, and that'd be more error prone and expensive to do at runtime than just making a simple trimmed down copy like this we can reuse as is 🙂